File: Modul-1/Soal-8/PRAK108-2410817220012-NOOR-KHALISA.c
#include <stdio.h>

// Definisi M_PI jika belum didefinisikan
#ifndef M_PI
#define M_PI 3.14159265358979323846
#endif

int main() {
    // Variabel
    int putaran = 5;
    float jarak = 14; // dalam kilometer

    // Keliling taman
    float kelilingTaman = jarak / putaran;

    // Jari-jari taman menggunakan rumus keliling
    float jariJari = kelilingTaman / (2 * M_PI);

    // Output
    printf("Diketahui:\n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", jarak);
    printf("\nJawaban:\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jariJari);

    return 0;
}

File: Modul-1/Soal-8/PRAK108-2410817220012-NOOR-KHALISA.py
import math

# Variabel
putaran = 5
jarak = 14 # dalam kilometer

# Keliling taman
keliling = jarak / putaran

# Jari-jari 
jari_jari = keliling / (2 * math.pi)

# Output
print("Diketahui:")
print(f"Pak Dengklek mengelilingi taman = {putaran} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak} Kilometer")
print("\nJawaban:")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari:.2f} Kilometer")